packages/services/f-statistics/CHANGELOG.md

```diff
@@ -3,6 +3,21 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.2.0
+------------------------------
+*August 09, 2021*
+
+- ### Fixed
+- Change build process to Vite
+
+
+v0.1.1
```

Comment:
Can this be part of v0.2.0 as the v0.1.1 won't be published separately?

Reply:
v0.1.1 was published; I needed to publish it prior to the PR to try and establish the issue